### PR TITLE
Docs sept2024 update

### PR DESCRIPTION
**Pull request by roscopeco**

### CODE REVIEW

**Review by 0xTJ — COMMENTED** [review]
Overall, looks good! I do notice one thing, the bottom of section 1 mentions contacting on Discord or email for the modification for non-USB UART use, but that's now documented in section 7.

**Comment by roscopeco** [comment]
> Overall, looks good! I do notice one thing, the bottom of section 1 mentions contacting on Discord or email for the modification for non-USB UART use, but that's now documented in section 7.

Ah good spot, thanks. Will fix that 👍

**Comment by roscopeco** [comment]
> Overall, looks good! I do notice one thing, the bottom of section 1 mentions contacting on Discord or email for the modification for non-USB UART use, but that's now documented in section 7.

Thanks again @0xTJ , this is now fixed 👍

File: docs/~$.2 Information.docx
Comment (0xTJ) [inline docs/~$.2 Information.docx:1]: The other fix to docs looks good! However it looks like you accidentally pushed up this temporary/lock(?) file. There's not much in it, but it probably shouldn't be kept in the repo.

Comment (roscopeco) [reply docs/~$.2 Information.docx:1]: Thanks again @0xTJ , fixed.

**Review by 0xTJ — APPROVED** [review]
LGTM! :rocket:

**Pull request merged**